app/models/P5/departure/TransportEquipment.scala

```diff
@@ -49,7 +49,7 @@ object TransportEquipment {
       case Some(firstElem :: Nil) => Some(s"${firstElem.sequenceNumber.getOrElse("")}:${firstElem.identifier.getOrElse("")}")
       case Some(firstElem :: tail) =>
         Some(
-          s"${firstElem.sequenceNumber.getOrElse("")}:${firstElem.identifier.getOrElse("")}...${tail.last.sequenceNumber.getOrElse("")}:${tail.last.identifier.getOrElse("")}"
+          s"${firstElem.sequenceNumber.getOrElse("")}:${firstElem.identifier.getOrElse("")}..."
```

Comment:
Possible refactoring opportunity here. See https://github.com/hmrc/transit-movements-trader-manage-documents/blob/main/app/models/P5/unloading/TransportEquipment.scala and https://github.com/hmrc/transit-movements-trader-manage-documents/blob/main/app/models/P5/package.scala

Reply:
[TAD_string - 2023-07-12T140759.892.pdf](https://github.com/hmrc/transit-movements-trader-manage-documents/files/12028367/TAD_string.-.2023-07-12T140759.892.pdf)

Reply:
TAD attached, for this PR "TRANSIT LIST OF ITEMS" section in TAD